sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -91,4 +93,21 @@ export class StudentController {
     // Save student
     return this.service.createStudent(userToken, payload);
   }
+
+  @Patch("/sync")
+  async synchronizeFromUserInfo(
+    @UserToken() userToken: IUserToken,
+  ): Promise<UserSyncInfoDto> {
+    const syncedUser = await this.studentService.synchronizeFromUserInfo(
+      userToken,
+    );
+
+    return {
+      firstName: syncedUser.user.firstName,
```

Comment:
Keeping the same behavior of returning the data that was already in place when this method was in the User endpoint.